ui/src/common/DateTimeSelector.tsx

```diff
@@ -9,22 +9,24 @@ interface DateTimeSelectorProps {
     showDate: boolean;
     label: string;
     popoverOpen?: (open: boolean) => void;
+    style?: object;
```

Comment:
This property is never set.

Reply:
Was added when I put margins on the time selector. I removed then when changing the icon to be on the left. It has been removed.